linux/Platform.c

```diff
@@ -287,8 +287,17 @@ void Platform_setMemoryValues(Meter* this) {
    long int usedMem = pl->usedMem;
    long int buffersMem = pl->buffersMem;
    long int cachedMem = pl->cachedMem;
-   usedMem -= buffersMem + cachedMem;
    this->total = pl->totalMem;
+
+   if (pl->settings->usedMemOnly) {
+      usedMem = pl->totalMem - lpl->availableMem;
+      this->values[0] = usedMem;
+      this->values[1] = 0;
+      this->values[2] = 0;
```

Comment:
Why not show buffersMem and cachedMem in the meter even with the changed usedMem calculation?

Reply:
Changed calculation resulted in higher memory consumption, so i thought that part of buffersMem and/or cachedMem is counted in usedMem. Such overlap might end up in showing more memory being used, than there actually is. But I might be just wrong. I asked for more info in linked issue.